README.md

```diff
@@ -124,6 +128,6 @@ If you run into trouble, please first refer to our [FAQ](https://ark-analysis.re
 If you would like to help make `ark` better, please take a look at our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html). 
 
 ## Citation
-Please cite our paper if you found our repo useful!
+Please cite our [paper](https://www.nature.com/articles/s41587-021-01094-0) if you found our repo useful! 
 
 [Greenwald, Miller et al. Whole-cell segmentation of tissue images with human-level performance using large-scale data annotation and deep learning](https://www.biorxiv.org/content/10.1101/2021.03.01.431313v2)
```

Comment:
You left in the old hyperlink. The published version should replaced the biorxiv link

Reply:
Oh my bad, I forgot that one.

Reply:
Or rather I updated the link in the wrong place.